pkg/gen/dependabot_ecosystem.go

```diff
@@ -9,14 +9,14 @@ import (
 const (
 	EcosystemDocker        Ecosystem = "docker"
 	EcosystemGithubActions Ecosystem = "github-actions"
-	EcosystemGo            Ecosystem = "go"
+	EcosystemGomod         Ecosystem = "gomod"
```

Comment:
I renamed that to match the real name. That way it will be easy to add new ecosystems.